### DISCUSSION TITLE
MIT License

### DISCUSSION THREAD

**Discussion by WindowToaster** [yasirkula/UnitySimpleFileBrowser#45] [Q&A] [answered]
I see that if I distribute the this as it is, I need to include the MIT License. I have a question, though? If I make a computer program in Unity3D that uses this as the file browser/saver/loader, but is not the main part of the program, do I need to include the MIT License here too, or can it be left out? If I do need to include the MIT License, where is it required (examples: within the user interface of the program itself, In the Unity3D editor before I build the program, As a text file that accompanies the program)?

**Comment by yasirkula** [comment, marked as answer]
I honestly don't know. I also ask myself the same questions sometimes. However, the asset is also available on Asset Store and AFAIK, you don't need to include a license file when you install the asset from Asset Store.

**Reply by WindowToaster** [reply]
Thank you so much for your response. The one I used was indeed the one from the Asset Store. I have, however, given you credit in the credits section of my computer program. Thank you so much for making this file browser!

**Reply by yasirkula** [reply]
I appreciate the credit, thank you 😸
